.github/workflows/test.yml

```diff
@@ -28,10 +28,34 @@ jobs:
           pip install pylint
           if [ -f setup.py ]; then pip install -e .; fi
```

Comment:
We can and probably should skip this here if we are creating a conda environment for slurm anyways.

Reply:
We could simply do all tests on a conda environment to avoid first installing mlagility outside the conda environment and then installing it again inside the conda environment. We can do that when/if testing times become an issue.